modules/mobile/app/workers/mobile/v0/vaccines_updater_job.rb

```diff
@@ -41,27 +40,26 @@ def perform
 
       def aggregate_source_data(aggregate, node)
         cvx_code = find_value(node, 'CVXCode')
-        group_name = find_value(node, 'Vaccine Group Name')
+        manufacturer = find_value(node, 'Manufacturer')
+
+        return nil if cvx_code.blank? || manufacturer.blank?
 
-        aggregate[cvx_code] = { group_names: [], manufacturer: nil } unless aggregate[cvx_code]
-        aggregate[cvx_code][:group_names] << group_name
-        aggregate[cvx_code][:manufacturer] = find_manufacturer(cvx_code) if group_name == 'COVID-19'
+        aggregate[cvx_code] = [] unless aggregate[cvx_code]
+        aggregate[cvx_code] << manufacturer
```

Comment:
capturing all manufacturers even though we currently only need the covid-19 ones

Reply:
Unless @kpethtel or @cadibemma have other ideas, I think that's fine. Might want to also double check with front-end to see if that would cause an issue.

Reply:
I agree that including manufacturer for all records is fundamentally better. It shouldn't be a problem here because 1) the adapter weeds out non-covid vaccines 2) all covid vaccines only have one manufacturer. 

If any covid vaccines ever have multiple manufacturers, it could cause some formatting issues. I don't think that will happen anytime soon, but it could be worth logging an error to sentry in the adapter if that ever occurs.

Reply:
It'd be worth logging an error if there's multiple covid manufacturers but that may get messy since some single covid manufacturers already have a comma in their name so in the adapter there's no fool proof way of knowing if the manufacturer string was one or more manufactures originally.